general/development/abc/faq.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Developer FAQ
+tags:
+  - FAQ
+  - Developers
+  - Getting started
+---
+
+<!-- cspell:ignore HRDNZ -->
+
+## Help for new coders
+
+### Where can I download Moodle?
+
+There are various ways you can download and install Moodle on your system.
+
+- Visit our [download page](https://download.moodle.org/)
+- Clone from [Moodle's Git repository](https://github.com/moodle/moodle) or find out more about using [Git for developers](https://docs.moodle.org/dev/Git_for_developers)
+
+### Where can I start?
+
+This very website is a great place to find out more how you can be a Moodle developer. Navigate the various sections to find out more.
+
+- [Guides](../../docs/) for Developer Documentation
+- [Community](../../docs/) for more information on how you can contribute
+- [Coding](../../docs/) for Moodle coding standards
+- [Process](../../docs/) to learn how the development cycle works
+
+#### Extend your learning
+
+- [Moodle Academy](https://moodle.academy/) online courses
+- MoodleBites for Developers online courses [Level 1](https://www.moodlebites.com/mod/page/view.php?id=24546) and [Level 2](https://www.moodlebites.com/mod/page/view.php?id=19542)
+
+### Who can I ask for help?
+
+The Moodle community is an active group of teachers and developers that have most likely encountered many of the same questions you might have. We encourage you to reach out and get involved in the available ways provided on our [Support channels](../../channels) page.
+
+### How do I create a patch?
+
+If you have made some changes to the code that you would like to share with the community, learn how to [create a patch](https://docs.moodle.org/dev/Git_for_developers#Preparing_a_patch).
+
+### How do I create a module or plugin?
+
+Use the following resources to help get you started with modules and plugins.
+
+- [Plugin tutorial](https://docs.moodle.org/dev/Tutorial)
+- [Blocks](/docs/apis/plugintypes/blocks)
+- [Authentication plugins](https://docs.moodle.org/dev/Authentication_plugins)
```

Comment:
```suggestion
```

Reply:
As mentioned before, I think Blocks and Authentication plugins can be removed from this list because both are included in the Plugin types.

Reply:
Hi @sarjona, So sorry I missed that one. I have fixed that up and pushed.